### PR TITLE
Add knucleotide4 to submitted graphs

### PR DESCRIPTION
**Pull request by bmcdonald3**
I had not added knucleotide4 to the graphs after adding my knucleotide4 submission to the submitted directory, this PR adds that to the graphs.

### CODE REVIEW

File: test/studies/shootout/submitted/knucleotide-submitted.graph [hunk 1]
```diff
@@ -1,5 +1,5 @@
 perfkeys: real, real
```
Comment (e-kayrakli) [inline test/studies/shootout/submitted/knucleotide-submitted.graph:1]: you need one more `, real` here

File: test/studies/shootout/submitted/knucleotide-submitted.graph [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
-graphkeys: release, submitted
-files: knucleotide.dat, knucleotide-submitted.dat
+graphkeys: release, submitted, knucleotide4
```
Comment (e-kayrakli) [inline test/studies/shootout/submitted/knucleotide-submitted.graph:2]: Should we give this a more descriptive name? I don't know enough about this test to say what's the distinguishing feature of this.

Comment (bmcdonald3) [reply test/studies/shootout/submitted/knucleotide-submitted.graph:2]: The submitted name is knucleotide4, but would you prefer something like `knucleotide-hash` instead? (the hash is the distinguishing feature)

Comment (e-kayrakli) [reply test/studies/shootout/submitted/knucleotide-submitted.graph:2]: If that's what sets this version apart from `release` and `submitted`, sure.

Comment (bmcdonald3) [reply test/studies/shootout/submitted/knucleotide-submitted.graph]: Alright, should be good to go

**Review by e-kayrakli — APPROVED** [review]

**Pull request merged**